test/api/importcsv/api_test.go

```diff
@@ -0,0 +1,114 @@
+package importcsv
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+func TestImportCSV(t *testing.T) {
+	for _, r := range TestCases {
+		t.Run(r.FileName, func(t *testing.T) {
+
+			// Upload CSV.
+			inputData := make(map[string]interface{})
+			err := Client.FilePost("/importsummaries/upload", r.FileName, &inputData)
+			if err != nil {
+				t.Errorf(err.Error())
+			}
+
+			// Check list of Applications.
+			importedApps, _ := Application.List()
+			expectedApps := r.ExpectedApplications
+			for i, importedApp := range importedApps {
+				if i >= len(expectedApps) {
+					t.Errorf("Extra imported Application: %s", importedApp.Name)
+					continue
+				}
+				expectedApp := expectedApps[i].Name
```

Comment:
The Name check on application is not enough, please check also other imported fields (if matches to expected values described in testcase/sample).

Reply:
Will do that! Also converted equality checks with assert.FlatEqual()